### ISSUE TITLE
support EXR images?

### ISSUE THREAD

**Issue by HAOHAOXUEXIEN** [srcn-ivl/UPR-Net#7]
can the UPR-Net support the exr input images? will  the  Effect deterioration? what should I do for support exr input， thanks very much.

**Comment by starrysky-ashin** [comment]
> can the UPR-Net support the exr input images? will the Effect deterioration? what should I do for support exr input， thanks very much.

Thanks for your interest on UPR-Net.

Our model is trained on RGB images, not EXR images. But, I think it also works for EXR images, if you convert EXR images to RGB as input. Actually, I don't know whether this will cause deteroration. You can have a try :)

**Comment by luuude** [comment]
Yes you just need to add openimageio and replace "cv2.imread" with "ImageInput.open" and the save for imwrite
